web/client/components/security/forms/PasswordReset.jsx

```diff
@@ -68,7 +61,7 @@ const PasswordReset = React.createClass({
           return null;
       }
       let pw = this.refs.password.getValue();
-      if (pw !== null && pw.length < this.props.minPasswordSize) {
+      if (pw !== null && pw.length < this.props.minPasswordSize && pw.length > 0) {
           return <Alert bsStyle="danger"><Message msgId="user.passwordMinlenght" msgParams={{minSize: this.props.minPasswordSize}}/></Alert>;
```

Comment:
i think is better to show the message when the user starts inserting something, or we can change the class from danger to info